apps/docs/src/app/platform/component-docs/platform-action-bar/platform-action-bar-header/platform-action-bar-header.component.html

```diff
@@ -1,8 +1,35 @@
 <header>Action Bar</header>
 <description>
-    The Action Bar is located at the top of the page and is used for Page title and Main Actions for the page.

```

Comment:
I think we should leave this sentence in. It gives users, who are not aware of the Core component, some context about what this component does.

Reply:
added sentence back.